gnomad_qc/v4/resources/release.py

```diff
@@ -162,9 +165,9 @@ def release_ht_path(
         if file_exists(public_release(data_type).versions[release_version].path):
             return public_release(data_type).versions[release_version].path
         else:
-            return f"gs://gnomad-public-requester-pays/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.sites.ht"
+            return f"gs://gnomad-public-requester-pays/release/{release_version}/ht/{data_type}/gnomad.{data_type}.v{release_version}.sites.filtered_combined_faf.ht"
```

Comment:
These are the paths that I wrote this out to, but we probably want to change this back to have it without that and just mv this to the other path